homeworks/01-bubble-sort/src/utils/arrayWithRandomValues.ts

```diff
@@ -0,0 +1,7 @@
+import { randomValue } from "./randomValueGenerator";
+
+export function arrayWithRandomValues(): number[] {
+  const lengthOfArray = Math.floor(randomValue(20, 40));
+
+  return Array.from({ length: lengthOfArray }).map(() => randomValue(1, 200));
```

Comment:
Just for yor information:
`Array.from({ length: lengthOfArray }, () => randomValue(1, 200))`